### DISCUSSION TITLE
DataProxy returned an error

### DISCUSSION THREAD

**Discussion by David81884** [ckan/ckan#7004] [Q&A]
Hi,
I'm new to CKAN.
I've searched for similar questions before posting this. Only one I can find was 7th April which didn't receive any responses so thought I would try asking a similar question.
We have a new installation of CKAN and I have uploaded a dataset. It's a simple spreadsheet with one row of just header names.
When I try to view it in Data Explorer I get:
**Could not load view: DataProxy returned an error (Data transformation failed. HTTPError: HTTP Error 404: Not Found)**

I can download the file no problem,
Can anyone give any pointers as to what the problem might be?
Thanks 
David

**Comment by duttonw** [comment]
Is this in relation to viewing the table on the page on a public/public dataset/resource or when it was set to private?
I do know that when datasets are private those api calls for the datastore are usually blocked (i know this is the case for data.qld.gov.au). 

Alternate questions if its related to pre-filling the datastore:
Is this via Xloader or DataPusher?
Did you configure an internal proxy to protect against metadata leaks from authors?
Is there a cache in the loopback that you are not bypassing?

If your so inclined, you can check out the opensource infra code of www.data.qld.gov.au at https://github.com/qld-gov-au/ckan-qld-infrastructure/blob/master/vars/shared-OpenData.var.yml 
for xloader configuration, this is located here:  https://github.com/qld-gov-au/opswx-ckan-cookbook/blob/master/recipes/ckanweb-deploy-exts.rb#L360 and https://github.com/qld-gov-au/opswx-ckan-cookbook/blob/656e60eef53b4788c4ccfbaead7cfdbd792050f2/templates/default/ckan_properties.ini.erb#L68

**Reply by David81884** [reply]
Hi William.
I've tried public and private datasets but both give the same error.
I'll need to refer the rest of these to our Python developer.
Thanks for your response.
Regards
David
